### DISCUSSION TITLE
Can Birdpi use a RPi alternative like a banana-pi?

### DISCUSSION THREAD

**Discussion by Billium28** [mcguirepr89/BirdNET-Pi#768] [General]
Due to the shortage of Raspberry Pis I want to know if one of the many similar boards can run this?

**Comment by jmtuts** [comment]
Have you looked at the Libre Computer Board AML-S905X-CC (Le Potato).  There is a whole discussion on this site showing how to make that work.   It is discussion #642

**Reply by Billium28** [reply]
Awesome, I'll give it a look thanks.
